### ISSUE TITLE
Add option to set the number for the of days of the periodicity of Asset Maintenance in Asset Module.

### ISSUE THREAD

**Issue by quebn** [frappe/erpnext#41725]
**No options to set a specific days of periodicity**
Currently the ERPNext Asset Maintenance only has the periodicity options of daily, weekly, monthly, and etc.
![image](https://github.com/frappe/erpnext/assets/93438531/faf7179e-7f07-4f55-b8ac-e1a554353323)
 but there are cases where some Companies' assets have a specific number of days like 10 days for example before it is due for another maintenance.

The option to set a specific days of periodicity should be available for these cases.

**Comment by Torchwood42** [comment]
Adding to this, even a period of 2 weeks would help me significantly. Optionality on this is disappointing and does not meet our business needs, ideally an option to select number of days would be possible as per issue subject. It would also be nice if asset maintenances to be assigned to a team rather than an individual, so that the load can be shared.

**Comment by Tropicalrambler** [comment]
I agree. I was searching today for a 10 year periodicity required for a specific asset.  I agree that the best way forward is to be able to select days between maintenance tasks instead, as well as the common ones:  daily, weekly, etc.
